### PR TITLE
Get orders trading page

### PR DESCRIPTION
**Pull request by bthaile**
https://github.com/AugurProject/augur/issues/2537

image

first step in getting orders on the market page. Here are open orders.

### CODE REVIEW

File: packages/augur-sdk/src/state/getter/Trading.ts [hunk 1]
```diff
@@ -297,8 +299,11 @@ export class Trading {
       const sharesEscrowed = convertOnChainAmountToDisplayAmount(new BigNumber(orderEventDoc.sharesEscrowed, 16), tickSize).toString(10);
       const tokensEscrowed = new BigNumber(orderEventDoc.tokensEscrowed, 16).dividedBy(10 ** 18).toString(10);
       let orderState = OrderState.OPEN;
-      if (amount === "0") {
-        orderState = orderEventDoc.eventType == 1 ? OrderState.CANCELED : OrderState.FILLED;
+      if (orderEventDoc.eventType == 3) {
```
Comment (justinbarry) [inline packages/augur-sdk/src/state/getter/Trading.ts]: Should this be an enum?

Comment (adrake33) [reply packages/augur-sdk/src/state/getter/Trading.ts]: Yep, it'd be better to use OrderEventType.Fill from augur-sdk/src/state/logs/types.ts here.

Comment (bthaile) [reply packages/augur-sdk/src/state/getter/Trading.ts]: @adrake33 can you go ahead and make that change on the PR?

Comment (adrake33) [reply packages/augur-sdk/src/state/getter/Trading.ts]: Sure, will do

**Review by adrake33 — APPROVED** [review]
Lgtm, but we'll need to work out which orders are returned by `getMarkets` as FILLED vs OPEN.

**Pull request merged**